core/src/test/scala/unit/kafka/admin/ListConsumerGroupTest.scala

```diff
@@ -29,7 +29,7 @@ import org.junit.jupiter.params.provider.ValueSource
 class ListConsumerGroupTest extends ConsumerGroupCommandTest {
 
   @ParameterizedTest
-  @ValueSource(strings = Array("zk", "kraft"))
+  @ValueSource(strings = Array("zk", "kraft", "kraft+kip848"))
   def testListConsumerGroups(quorum: String): Unit = {
```

Comment:
It doesn't work for kraft+kip848. If changing GroupCoordinatorService#ListGroups with the same way of adding the futures as describeGroups, this test passes. Not sure why